test/SecTester.Scan.Tests/Mocks/MockLogger.cs

```diff
@@ -0,0 +1,19 @@
+namespace SecTester.Scan.Tests.Mocks;
+
+internal abstract class MockLogger : ILogger
```

Comment:
There are a lot of duplicates from your previous PR. How do you want me to review them? https://github.com/NeuraLegion/sectester-net/pull/76#discussion_r1038232222